.travis.yml

```diff
@@ -25,6 +25,16 @@ matrix:
     - name: "ARCH=x86_64 REPO=linux-next"
       env: ARCH=x86_64 REPO=linux-next
       if: type = cron
+    # stable
+    # TODO: make cron only
+    - name: "ARCH=arm64 REPO=4.18"
+      env: ARCH=arm64 REPO="4.18"
```

Comment:
strange, the logs show the default value being used below, as if this is not set.